.flexci/linux/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -uex
+
+apt update
+apt install -y git curl wget gcc make openssl libssl-dev libbz2-dev libreadline-dev libsqlite3-dev libffi-dev zlib1g-dev liblzma-dev
+
+git clone https://github.com/pyenv/pyenv.git /opt/pyenv
+PYENV_ROOT=/opt/pyenv
```

Comment:
export 書かないと実質的に効果がないはず。

```suggestion
export PYENV_ROOT=/opt/pyenv
```